packages/protocol/contracts/stability/MedianDeltaBreaker.sol

```diff
@@ -117,14 +122,17 @@ contract MedianDeltaBreaker is IBreaker, UsingRegistry {
    * @notice Checks if the specified current median rate is within the allowed threshold.
    * @param lastRate The last median rate.
    * @param currentRate The current median rate.
-   * @return  Returns a bool indicating whether or not the current rate 
+   * @return  Returns a bool indicating whether or not the current rate
    *          is within the given threshold.
    */
   function isWithinThreshold(uint256 lastRate, uint256 currentRate, uint256 allowedThreshold)
     private
     pure
     returns (bool)
   {
+    // uint256 maxPercent = FixidityLib.newFixed(1).add(allowedThreshold);
+    // uint256 maxValue = (FixidityLib.newFixed(lastRate).multiply(maxPercent)).fromFixed();
+
     uint256 maxPercent = uint256(1 * 10**24).add(allowedThreshold);
```

Comment:
Use FixidityLib here, i.e. grab `fixed1()` instead of `10**24`, pass in a FixidityLib fraction as argument.

Reply:
I'm against using Fixidity here. It doesn't provide any benefits over the current implementation and costs 3x more. Will add Fixed1 and unwrap.
![Screenshot from 2022-07-28 11-16-06](https://user-images.githubusercontent.com/6872903/181587739-1d14c85e-b04f-4d15-b5a9-907a5f00a1fa.png)

Reply:
The benefit of using Fixidity is it helps keep the code clean, without needing to remember the details of the specific fixed point arithmetic we're using here, makes sure all the arithmetic is being done exactly the same throughout the code base, and updates to the lib get propagated to all uses of it.

The cost difference is pretty big (any idea why, btw? is this just from all the `delegatecall`s to a linked library?) but we need to consider the trade-off with maintainability of the code.